airflow/providers/common/sql/hooks/sql.py

```diff
@@ -294,6 +297,22 @@ def _run_command(self, cur, sql_statement, parameters):
         if cur.rowcount >= 0:
             self.log.info("Rows affected: %s", cur.rowcount)
 
+    def _update_query_ids(self, cursor) -> None:
+        """
+        Adds query ids to list
+        :param cur: current cursor after run
+        :return:
+        """
+        return None
+
+    def stop_query(self, query_id) -> Any:
+        """
+        Stops query with certain identifier
+        :param query_id: identifier of the query
+        :return:
+        """
+        return None
```

Comment:
I know we debated this on slack but we should also get some resolution here...
should we return None or `NotImplementedError`?
we can catch the NotImplementedError in the operator and convert it to INFO log entry in the log notifying users that their request to kill the query can not be processed due to lack of implementation.

Reply:
That's a good idea in my opinion.
Because I assume that if the query is already completed and we try to kill it, Snowflake might raise an error.